client/src/components/table/component.tsx

```diff
@@ -12,7 +12,7 @@ import Cell, { HeaderCell } from './cell';
 import type { ColumnDefinition } from './column';
 import TableRow, { TableHeaderRow } from './row';
 
-export interface TableProps<T = unknown>
```

Comment:
ensure @andresgnlez's PR doesn't have a conflict here because he was changing this files. Anyway this change couldn't be part of the feature